drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -154,6 +154,8 @@ def self.authorized?(object, ctx)
       composite_perm :can_upload_client_files, permissions: [:manage_any_client_files, :manage_own_client_files], mode: :any
       composite_perm :can_view_any_files, permissions: [:manage_own_client_files, :view_any_nonconfidential_client_files, :view_any_confidential_client_files], mode: :any
       can :audit_clients
+      can :view_client_alerts
```

Comment:
There must be a command I need to run to get the schema.graphql file to update, right?